content/develop/connect/insight/rdi-connector.md

```diff
@@ -0,0 +1,155 @@
+
+```
+├── config.yaml
+└── jobs
+    └── job1.yaml
+```
+
+The `config.yaml` file, your configuration pipeline YAML file, is required. The `jobs` directory can be empty, as job pipelines are not required, but the empty directory must exist in the zip file. Otherwise, the `jobs` folder might include one or more transformation job pipeline files.
+
+### Create a new configuration pipeline using the built-in editor
+
+Click the **Create new pipeline** button in the **Start with your pipeline** dialog to create new pipeline data using Redis Insight's built-in editor. After doing so, you'll enter the **Configuration file** editor and you'll see a **Select a template** dialog in the upper right-hand corner of the editor.
+
+<img src="../images/ri-rdi-pl-editor-dlg.png">
+
+Make your selections in the provided fields:
+
+- **Pipeline type** is set to **Ingest**. This is the only available option.
+- **Database type** has six options:
+  - mongodb
+  - cassandra
+  - mysql
+  - oracle
+  - postgresql
+  - sqlserver
+
+After you make your selections and click **Apply**, Redis Insight will populate the editor window with an appropriate template. To start from scratch, click **Cancel**.
+
+<img src="../images/ri-rdi-pl-editor-template.png">
+
+### Create a new transformation job pipeline using the built-in editor
+
+In the **Pipeline Management** pane, click the `+` next to the **Jobs** folder and enter a name for the new job.
+Next, click on the job name you just created.
+This will place you in the transformation job editor with the template selection menu open. There's only one template available: **Ingest**. Click **Apply** and Redis Insight will populate the editor windows with an appropriate template. To start from scratch, click **Cancel**.
+
+<img src="../images/ri-rdi-job-editor-template.png">
+
+{{< warning >}}
+Any changes you make in the editors will be lost if you exit Redis Insight without saving your work. To save any changes you made to your pipeline files, deploy them to your RDI server (see below) or download the modified files as a zip file to your local disk using the **Download** button in the top right of the RDI window.
+{{< /warning >}}
+
+## The built-in editors
+
+The Redis Insight pipeline editors are context-aware. They provide auto-completion and syntax highlighting for:
+
+- YAML files in the configuration and job file editors
+- JMESPath and SQL function snippets in a dedicated editor. To open the JMESPath and SQL editor, click the **SQL and JMESPathEditor** button as shown above. A new editor window will open in the lower half of the screen.
+
+For example, if you decided to write your own configuration pipeline without using a template, you would see auto-completion prompts such as the one shown below.
+
+<img src="../images/ri-rdi-pl-editor-autoc.png">
+
+While this isn't a replacement for the RDI documentation, it can help speed you along once you have basic familiarity with the building blocks of RDI pipeline files.
```

Comment:
```suggestion
While this isn't a replacement for the RDI documentation, it can help speed you along after you have basic familiarity with the building blocks of RDI pipeline files.
```